kube/src/lib.rs

```diff
@@ -168,6 +168,10 @@ pub use kube_derive::CustomResource;
 #[cfg_attr(docsrs, doc(cfg(feature = "derive")))]
 pub use kube_derive::Resource;
 
+#[cfg(feature = "derive")]
+#[cfg_attr(docsrs, doc(cfg(feature = "derive")))]
+pub use kube_derive::Validated;
```

Comment:
naming wise, possibly this introduces an confusion possiblities with `garde::Validate` - which we do advocate for